src/SFA.DAS.Admin.Aan.Web/Views/NetworkEvent/EventDescription.cshtml

```diff
@@ -0,0 +1,113 @@
+                <div esfa-validation-marker-for="@Model.EventOutline" class="govuk-form-group"> 
+                    <span asp-validation-for="@Model.EventOutline" class="govuk-error-message"></span>
+                    <div id="EventOutline">
+                        <h1 class="govuk-label-wrapper">
+                            <label class="govuk-label govuk-label--m" for="eventOutline">
+                               Event outline
+                            </label>
+                        </h1>
+                        <div id="eventOutline-hint" class="govuk-hint">
+                            Short description that outlines the event listing
+                        </div>
+                        <textarea class="govuk-textarea govuk-js-character-count" id="eventOutline" name="eventOutline" rows="5" aria-describedby="eventOutline-info eventOutline-hint"
+                                      sfa-validationerror-class="form-control-error" maxlength="500">@Model.EventOutline</textarea>
+                    </div>
+                    <div id="eventOutline-info" class="govuk-hint govuk-character-count__message">
+                        You can enter up to @Model.EventOutlineMaxCount characters
+                    </div>
+                </div>
+            </div>
+
+            <div class="govuk-character-count" data-module="govuk-character-count" data-maxlength="@Model.EventSummaryMaxCount">
+                <div esfa-validation-marker-for="@Model.EventSummary" class="govuk-form-group"> 
+                    <span asp-validation-for="@Model.EventSummary" class="govuk-error-message"></span>
+                    <div id="EventSummary">
+                        <h1 class="govuk-label-wrapper">
+                            <label class="govuk-label govuk-label--m" for="eventSummary">
+                                Event summary
+                            </label>
+                        </h1>
+                        <div id="eventSummary-hint" class="govuk-hint">
+                            A summary of the event including an agenda and any pre-event preparation required.
+                            Include any practical information such as how to enter the venue, parking and nearest bus/tram/tube/train stop to the venue.
+                        </div>
+                        <div class="app-editor">
+                            <textarea class="govuk-textarea govuk-js-character-count" id="eventSummary" name="eventSummary" rows="5" aria-describedby="eventSummary-info eventSummary-hint"
+                                      sfa-validationerror-class="form-control-error" maxlength="2500">@Model.EventSummary</textarea>
+                        </div>
+                    </div>
+                    <div id="eventSummary-info" class="govuk-hint">
+                        You can enter up to @Model.EventOutlineMaxCount characters
```

Comment:
EventSummaryMaxCount  should be used instead of EventOutlineMaxCount

Reply:
ok thanks, fixed